README.md

```diff
@@ -188,8 +199,15 @@ from luxtronik import Luxtronik, Parameters
 
 l = Luxtronik('192.168.1.23', 8889)
 
+# Queue a parameter change
+l.parameters.set("ID_Einst_Heizgrenze", 14.0)
```

Comment:
This parameter is not marked as `safe` to write, and I think that `14.0` is not a valid value. I suggest to use some parameter, which is actually useful (together with a short comment), what this parameter is about, e.g., `ID_Einst_WK_akt` (temperature offset for heating), `ID_Soll_BWS_akt` (DHW temperature) or `ID_Einst_BWS_Hyst_akt` (DHW hysteresis).

Reply:
Now use `ID_Soll_BWS_akt`